<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-HE9LB5U" failures="1" tests="1" name="runner.CucumberRunnerTests" time="3.266" errors="0" timestamp="2022-09-21T11:13:17 MSK" skipped="0">
  <testcase classname="runner.CucumberRunnerTests" name="runScenario" time="3.266">
    <failure message="data[0].avatar
Expected: hhttps://reqres.in/img/faces/7-image.jpg
     got: https://reqres.in/img/faces/7-image.jpg
" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: data[0].avatar
Expected: hhttps://reqres.in/img/faces/7-image.jpg
     got: https://reqres.in/img/faces/7-image.jpg

at org.skyscreamer.jsonassert.JSONAssert.assertEquals(JSONAssert.java:417)
at org.skyscreamer.jsonassert.JSONAssert.assertEquals(JSONAssert.java:394)
at definitions.StepDefinitions.positiveTest(StepDefinitions.java:28)
at âœ½.I get forecast(file:///C:/Users/lexey/IdeaProjects/Bolshakov_Task2/src/test/resources/features/PositiveTest.feature:4)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- runner.CucumberRunnerTests -->
